command/base64/base64.go

```diff
@@ -0,0 +1,136 @@
+package base64
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/cli/command"
+	"github.com/smallstep/cli/utils"
+	"github.com/urfave/cli"
+)
+
+func init() {
+	cmd := cli.Command{
+		Name:      "base64",
+		Action:    command.ActionFunc(base64Action),
+		Usage:     "encodes and decodes b64 data.",
+		UsageText: `**step base64** [**-d**|**--decode**] [**-r**|**--raw**] [**-u**|**--url**]`,
+		Description: `**step base64** implements base64 encoding as specified by RFC 4648.
+
+## Examples
+
+Encode to base64 using the standard encoding:
+'''
+$ echo -n This is the string to encode | step base64
+VGhpcyBpcyB0aGUgc3RyaW5nIHRvIGVuY29kZQ==
+$ step base64 This is the string to encode 
+VGhpcyBpcyB0aGUgc3RyaW5nIHRvIGVuY29kZQ==
+'''
+
+Decode a base64 encoded string:
+'''
+$ echo VGhpcyBpcyB0aGUgc3RyaW5nIHRvIGVuY29kZQ== | step base64 -d
+This is the string to encode
+'''
+
+Encode to base64 without padding:
+'''
+$ echo -n This is the string to encode | step base64 -r
+VGhpcyBpcyB0aGUgc3RyaW5nIHRvIGVuY29kZQ
+$ step base64 -r This is the string to encode
+VGhpcyBpcyB0aGUgc3RyaW5nIHRvIGVuY29kZQ
+'''
+
+Encode to base64 using the url encoding:
+'''
+$ echo 'abc123$%^&*()_+-=~' | step base64 -u
+YWJjMTIzJCVeJiooKV8rLT1-Cg==
+'''
+
+Decode an url encoded base64 string:
```

Comment:
should these be equivalent? weird example to show if they are both the same.

Reply:
The second one with the `-u` flag enforces the URL encoding. Without it will autodetect it. 
We can get rid of the second example if we want, the most common case should be without the `-u`.

Reply:
sure, I just mean that based on the examples it appears that the two don't actually do anything different. which is probably not what you're trying to show.

Reply:
I'll add a comment.